pygfx/utils/transform.py

```diff
@@ -571,36 +583,80 @@ def __init__(
         self._scale_view = self._scale.view()
         self._scale_view.flags.writeable = False
 
+        if matrix is not None:
```

Comment:
Ah I was confused by the use of `._matrix`, and had to closely examine the code to see how it's used. Let's make it explicit for future maintenance.
```suggestion
        # The ._matrix is only used when state_basis is "matrix"
        if state_basis == "matrix" and matrix is not None:
```